.github/workflows/stale.yml

```diff
@@ -6,12 +6,12 @@ on:
     - cron: '0 12 * * *'
 
 permissions:
-  issues: write
   pull-requests: write
 
 env:
-  BEFORE_ISSUE_STALE: 90
-  BEFORE_ISSUE_CLOSE: 7
+  # NOTE: We set the parameters below to -1, so issues will never be marked as stale or closed automatically.
+  BEFORE_ISSUE_STALE: -1
+  BEFORE_ISSUE_CLOSE: -1
```

Comment:
Let's just remove the bits related to issues. I don't think we'll need that again. And if we do, we can still recover it from history.

Reply:
Sadly, it seems it doesn’t work that way. The action actually creates a default value unless issues are explicitly excluded. See: https://github.com/actions/stale/issues/1112#issuecomment-1963669908 and https://github.com/actions/stale/blob/f69122271d990fd11f5594ccff2296f00ff59b49/action.yml#L21.

To address this, I set the issue-related setting to `-1` and removed any settings that are no longer relevant.